tools/scripts/ditslint.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/bash
 dtslint --expectOnly --localTs node_modules/typescript/lib types
+npm run lint
```

Comment:
I'm not sure we want to run "npm run lint" after every single command.

I think we should keep it in "npm run test" and "npm run compile/build", All other commands are used within these two. What do you think?